src/components/TokenEnterAmount.tsx

```diff
@@ -179,7 +179,8 @@ export function useEnterAmount(props: {
             .replace(new RegExp(`[${decimalSeparator}]?0+$`), '')
         : ''
 
-    const parsedTokenAmount = parseInputAmount(convertedLocalToToken, decimalSeparator)
+    const parsedTokenAmount =
+      amount === '' ? null : parseInputAmount(convertedLocalToToken, decimalSeparator)
```

Comment:
parseInputAmount always returns a big number and returns 0 if the convertedLocalToToken (derived from  `amount`) is an empty string